examples/dictionary-example.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+require "vendor/autoload.php";
+
+use Momento\Auth\EnvMomentoTokenProvider;
+use Momento\Cache\SimpleCacheClient;
+
+$MOMENTO_AUTH_TOKEN = getenv("MOMENTO_AUTH_TOKEN");
+$CACHE_NAME = getenv("CACHE_NAME");
+$DICTIONARY_NAME = "exmaple-dictionary";
+$FIELD = "MyField";
+$VALUE = "MyValue";
+$ITEM_DEFAULT_TTL_SECONDS = 60;
+
+function printBanner(string $message): void
+{
+    $line = "******************************************************************";
+    print "$line\n$message\n$line\n";
+}
+
+printBanner("*                      Momento Example Start                     *");
+
+// Setup
+$authProvider = new EnvMomentoTokenProvider("MOMENTO_AUTH_TOKEN");
+$client = new SimpleCacheClient($authProvider, $ITEM_DEFAULT_TTL_SECONDS);
+
+// Ensure test cache exists
+$response = $client->createCache($CACHE_NAME);
+if ($response->asSuccess()) {
+    print "Created cache " . $CACHE_NAME . "\n";
+}
+if ($response->asAlreadyExists()) {
+    print "Cache " . $CACHE_NAME . " already exists.\n";
+}
+if ($response->asError()) {
+    print "Error creating cache: " . $response->asError()->message() . "\n";
+    exit;
+}
```

Comment:
This might be better as a single if/elseif rather than three separate if statements since the response can only be one of the three types.